leshan-core/src/test/java/org/eclipse/leshan/core/node/codec/LwM2mNodeEncoderTest.java

```diff
@@ -189,7 +189,7 @@ public void json_encode_device_object_instance() {
         b.append("{\"n\":\"9\",\"v\":100},");
         b.append("{\"n\":\"10\",\"v\":15},");
         b.append("{\"n\":\"11\",\"v\":0},");
-        b.append("{\"n\":\"13\",\"v\":1367491215},");
+        b.append("{\"n\":\"13\",\"v\":1.367491215E9},");
```

Comment:
Don't we want to keep the value without exponent?

Reply:
I agree this is strange to encode a date with this notation... But I think this respect the specification as in JSON, there is only one format for number called "Number" :p (see https://json.org/)

**Why this behave like this :**
When we encode we use several format :
LwM2mNode > LWM2M JSON  > JSON > String

The resource 13 is a date but we loose this information during conversion.
Date ==(leshan)==> Number ==(leshan)==> Double ==(minimal-json)==> String

Minimal-json use [Double.toString](https://docs.oracle.com/javase/7/docs/api/java/lang/Double.html#toString(double))

We can imagine to use a custom minimal-json JsonWriter (it sounds not so good)
or trying to no convert all Number in Double. (we need to add some instanceof in JsonArrayEntrySerDes.jSerialize)

I don't know if it worth it?

Reply:
Ok. indeed, the scientific notation is valid in Json. Let's keep it that way :)